enterprise/server/util/vsock/vsock.go

```diff
@@ -132,3 +136,42 @@ func DialHostToGuest(ctx context.Context, socketPath string, port uint32) (net.C
 	}
 	return conn, nil
 }
+
+// SimpleGRPCDial internally calls DialHostToGuest and then sets up a gRPC
```

Comment:
could `DialHostToGuest` be made private now or is it still used elsewhere?

Reply:
Done.